docs/guidelines/accessibility.md

```diff
@@ -1,3 +1,659 @@
+###### Non-Literal Text
+
+Non-literal text, such as sarcasm, satire, parody, allegory, metaphor, slang, and colloquialisms, can be a problem for some readers.
+
+###### Non-Existent Text
+
+The unstated assumptions and implied meaning of written content may seem obvious to the writer, but readers may not have the necessary background knowledge. Some readers may not have the skills to infer meaning from text without additional help.
+
+##### Obstacle 5: Math comprehension
+
+Mathematical expressions are not easy for everybody to understand. 
+
+For people who are comfortable reading equations and thinking mathematically, the best way to explain mathematical concepts is to use equations. On the other hand, often it is helpful to explain math conceptually, either with or without the formulas. 
+
+Conceptual explanations help readers understand the reasoning behind the math.
+
+##### Obstacle 6: Visual comprehension
+
+Some individuals have difficulties processing visual information, the opposite of the problem experienced by people with reading and verbal processing difficulties. 
+
+They may recognize the fact that there are objects on a web page, but may not be able to identify the objects. 
+
+For example, they may not realize that a photograph of a person is a representation of a person, though they can plainly see the photograph itself (as an object) on the web page.
+
+For these people, a moving, talking person in a video may be easier to identify and mentally process than a static image of a person in a photograph. Video and multimedia, accompanied with narration, may be the best way to communicate to these individuals.
+
+#### Seizure
+
+Some people are susceptible to seizures caused by **strobing**, **flickering**, or **flashing** effects. This kind of seizure is sometimes referred to as a photoepileptic seizure because it is caused by pulses of light (hence the prefix "photo") interacting with the eye's light-receptive neurons and the body's central nervous system.
+
+Most web content is completely harmless to individuals with photoepileptic tendencies. Even most animations, videos, moving text, and Flash objects do not present any danger. 
+
+We should still make every effort to ensure that our content does not have strobing, flickering, or flashing effects.
+
+#### Vestibular disorders
+
+Even if an animating or moving object does not cause a seizure, it may cause nausea or dizziness in some people.
+
+The following items can result in difficulties for users with vestibular disorders:
+
+- High contrast graphics with tight parallel lines.
+- Animated scrolling that lasts longer than perhaps 1/4 second.
+- Parallax or reverse parallax - simultaneous foreground and background scrolling in different directions or at different speeds.
+- Moving images beneath static text.
+
+
+Resources:
+
+- [W3C:  How People with Disabilities Use the Web](https://www.w3.org/WAI/people-use-web/)
+
+
+
+
+
+## What you can do to make our website accessible
```

Comment:
this list could potentially be broken part into a different page. This is a good overview of actionable items/checklist

Reply:
I don't think we get into too much details of how to implement accessibility in the global overview/info area.  I'm thinking to cover some high level stuff and add some external resource links for more detailed info.  I'm thinking we get into details in Mayflower specific stuff, so we can express our implementation details and why there.  Let's talk about it more!